episodes/simpleitk.md

```diff
@@ -935,6 +935,10 @@ TBD
 
 This episode was largely inspired by [the official SITK tutorial](https://simpleitk.org/TUTORIAL/#tutorial), which is copyrighted by NumFOCUS and distributed under the [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/).
 
+## Acknowledgements
```

Comment:
This is repeated twice, not sure what happened but I see you added this section @drcandacemakedamoore